docs/en/03_How_tos/redirections.md

```diff
@@ -14,7 +14,7 @@ Implementing in `.htaccess` is more performant than using PHP to redirect and al
 
 ```
 RewriteCond %{HTTP_HOST} ^redirection\.com(.*)$ [NC]
-RewriteRule ^(.*)$ http://www\.redirection\.com/$1 [R=301,L]
+RewriteRule ^(.*)$ http://www\.redirection\.com/$1 [R=302,L]
```

Comment:
Could you please update with a note as to why people should be careful and use `302 Moved Temporarily` rather than `301`?
Perhaps a note that once tested it could be updated... 🤷‍♂️